gpkit/__init__.py

```diff
@@ -121,7 +120,7 @@ def _subvert_pint():
         When gpkit objects appear in mathematical operations with pint
         Quantity objects, let the gpkit implementations determine what to do
         """
-        def skip_if_gpkit_objects(fallback, objects=(PosyArray, Signomial)):
+        def skip_if_gpkit_objects(fallback, objects=(NomialArray, Signomial)):
```

Comment:
I see you changed to `Nomial`, I like it. If we're worried about the import in `__init__` I suppose we could implement `_subvert_pint` in a separate file. But I don't personally have a problem with importing `Nomial`.